data.sql

```diff
@@ -11,3 +11,30 @@ VALUES ( 'Pikachu', '2021-01-07', 1 , '0', 15.04);
 
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
 VALUES ( 'Devimon', '2017-05-12', 5 , '1', 11.0);
+
+
+
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ( 'Charmander', '2020-02-08', 0 , '0' , -11.0);
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ( 'Plantmon', '2021-11-15', 2 , '1' , -5.7);
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ( 'Squirtle', '1993-04-02', 3 , '0' , -12.13);
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ( 'Angemon', '2005-06-12', 1 , '1' , -45.0);
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ( 'Boarmon', '2005-06-07', 7 , '1' , 20.4);
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ( 'Blossom', '1998-10-13', 3 , '1' , 17.0);
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ( 'Ditto', '2022-05-14', 4 , '1' , 22.0);
```

Comment:
- [ ] it would be better to change the `neutered` from int to boolean and use true and false that will be close to the requirements